File: practice_java_program/notpad/CountFrequency.java
import java.util.*;
public class CountFrequency{
	public static void main(String[] args){
		
		String name = "Neeraj Singh";
		HashMap<Character, Integer> hs= new HashMap<>();
		
		for(char ch : name.toCharArray()){
			if(ch==' '){
				continue;
			}
			else{
				hs.put(ch, hs.getOrDefault(ch, 0)+1);
			}
		}
		for(char ch:hs.keySet()){
			System.out.println(ch +" = " +hs.get(ch));
		}
	}
}

File: python_oriented_interview/notpad/CountFrequency.py
name = "Neeraj Singh"
hs = {}

for ch in name:
    if ch == ' ':
        continue
    else:
        hs[ch] = hs.get(ch, 0) + 1

for ch in hs:
    print(f"{ch} = {hs[ch]}")
